### DISCUSSION TITLE
React flow Node clickable pointer vs hover pointer

### DISCUSSION THREAD

**Discussion by Krithika3** [xyflow/xyflow#2173] [General]
First of all I would like to say that we love react flow and we have been using this for graphs in our apps. One question I have is our Nodes are fairly static, and I noticed that we have a hover pointer when I go over the node vs a finger pointer that indicates a click (since my graph uses the onNodeClick prop on React flow). Any inputs/advise on changing it to the finger pointer to indicate it is clickable?

**Comment by Krithika3** [comment]
I think I can just the cursor pointer for nodes, but this surprisingly does not work for the edges

**Reply by jakemager** [reply]
Strange for me the pointer is very easy to change for edges but I can't get it change for custom nodes for the life of me. I was able to change it for built in nodes by manipulating the css class

**Comment by Tarun-mymobioffice** [comment]
I loved your name @Krithika3 , genuinely.Also thnx for resolving the doubt
